pgmpy/tests/test_models/test_JunctionTree.py

```diff
@@ -38,3 +40,43 @@ def test_add_cyclic_path_raises_error(self):
 
     def tearDown(self):
         del self.graph
+
+class TestJunctionTreeCopy(unittest.TestCase):
+    def setUp(self):
+        self.graph = JunctionTree()
+
+    def test_copy(self):
+        self.graph.add_nodes_from([('a', 'b', 'c'), ('a', 'b') ,('a', 'c')])
+        self.graph.add_edges_from([(('a', 'b', 'c'), ('a', 'b')),
+                                   (('a', 'b', 'c'), ('a', 'c'))])
+        graph_copy = self.graph.copy()
+        self.graph.remove_edge(('a', 'b', 'c'), ('a', 'c'))
+        self.assertFalse(self.graph.has_edge(('a', 'b', 'c'), ('a', 'c')))
+        self.assertTrue(graph_copy.has_edge(('a', 'b', 'c'), ('a', 'c')))
+        self.graph.remove_node(('a', 'c'))
```

Comment:
@Pratik151 Add a few blank lines in this method to group things together. 
For example in lines 53-55,  you remove some edges and test it so keep them together and add a blank line after line 55.